bundles/org.openhab.binding.dmx/README.md

```diff
@@ -194,7 +194,7 @@ The next `ON` command uses these values instead of the default (or configuration
 |-----------------|---------------------|----------------------|----------------------------------------------------|
 |brightness       |dimmer, tunablewhite |Switch, Dimmer        | controls the brightness                            |
 |color            |color                |Switch, Dimmer, Color | allows to set the color and brightness             |
-|colortemperature |tunablewhite         |Number                | allows to set the color temperature                |
+|color_temperature |tunablewhite         |Number                | allows to set the color temperature                |
```

Comment:
```suggestion
|color_temperature|tunablewhite         |Number                | allows to set the color temperature                |
```

Reply:
@funkstille Could you accept @J-N-K's change suggestion? We could then merge. Thanks!